examples/fit_with_bounds.py

```diff
@@ -61,58 +73,85 @@ def gen_from_expr(self,data, guesses={}):
                 "axis names are",
                 axis_names,
                 "variable names are",
-                variable_names,
+                self.variable_names,
                 "parameter names are",
                 parameter_names,
             )
         )
         # }}}
-        pars = Parameters()
-        for this_name in parameter_names:
-            kwargs = {}
-            if this_name in guesses.keys():
-                logger.debug(strm("applying bounds for", this_name))
-                kwargs.update(guesses[str(this_name)])
-            pars.add(this_name, **kwargs)
-        for j in pars:
-            logger.info(strm("fit param ---", j))
-        logger.info(strm(pars))
         self.fn = lambdify(
```

Comment:
will this placement be okay? (before we declared Parameters prior to lambdify and now we declare Parameters after)

Reply:
The order of the methods inside the class definition has no meaning.